src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ManagedGroupServiceSpec.scala

```diff
@@ -40,7 +40,7 @@ class ManagedGroupServiceSpec extends FlatSpec with Matchers with TestSupport wi
   private val resourceActionPatterns = resourceActions.map(action => ResourceActionPattern(action.value))
   private val defaultOwnerRole = ResourceRole(ManagedGroupService.adminRoleName, resourceActions)
   private val defaultRoles = Set(defaultOwnerRole, ResourceRole(ManagedGroupService.memberRoleName, Set.empty))
-  private val managedGroupResourceType = ResourceType(ManagedGroupService.managedGroupTypeName, resourceActionPatterns, defaultRoles, ManagedGroupService.adminRoleName)
+  private val managedGroupResourceType = ResourceType(ManagedGroupService.managedGroupTypeName, resourceActionPatterns, defaultRoles, ManagedGroupService.adminRoleName, true)
```

Comment:
should this be read from config?

Reply:
Yeah, exactly what I was trying to reference in my PR comment.  I would argue yes, it should be pulling from the config.  But this is also a bigger change, which is why I didn't really get into too much detail about it.  

Maybe for this PR we just consider changing the ManagedGroup*Specs and make sure they're pulling from the config file.  

As for the other ResourceTypes, we don't really test them specifically, which I think is a hole in our testing.  That config file is constructed in a careful way, and messing it up could be problematic.  I suppose the automation suite might eventually put that config to test, but I don't know how completely it does so, and that's also pretty far down the process pipeline before we're actually testing.  

On the other hand...the configuration of Resource Types probably doesn't change that often, so is it worth having the testing overhead?  (I would argue that it is worth it)

Managed Groups are the oddballs in the list of Resource Types because they have a bunch of code that's specific to them, which is unusual in SAM.  So modifying the ManagedGroup*Specs to load the resource type from the configs probably makes sense.  

As for the other resource types, if we wanted to test them, I think we would make a specific test suite that simply asserts that we have the policies and actions correctly set up for each resource type.  

@MatthewBemis I think we should face talk about this some more.

Reply:
I agree that they should be read from config